src/components/StartButton.js

```diff
@@ -1,42 +1,56 @@
 import '../css/App.css';
-import newQuote from '../utils/quoteGenerator';
 
-const StartButton = ({currentQuoteProp, setQuoteProp, setAuthorProp, buttonLabelProp, gameStateProp, setGameStateProp, greenButtonProp, setGreenButtonProp, redButtonProp, setRedButtonProp}) => {
-
-    const handleClick = (event) => {
-        let quoteObject;
-        let quoteText;
-        let quoteAuthor;
-        do {
-            quoteObject = newQuote();
-            quoteText = quoteObject.quotation;
-            quoteAuthor = quoteObject.author;
-        } while (currentQuoteProp === quoteText); 
-        setQuoteProp(quoteText);
-        setAuthorProp(quoteAuthor);
-
-        document.getElementById("author-label").textContent = "";
-
-        if (greenButtonProp === "CORRECT!" || "WRONG!") {
-            setGreenButtonProp("It's Kev!")  
-        };
-        if (redButtonProp === "CORRECT!" || "WRONG!") {
-            setRedButtonProp("Not Kev!")        
-        };
-
-        setGameStateProp("open");
+const StartButton = ({statusProp, clickHandler, resetClickHandler}) => {
+    let buttonText;
+    switch(statusProp) {
```

Comment:
Could this be done the same way as the `it's kev` button? Where the markup is only created once?

Reply:
Nice one - done that in 78a8c90 !